src/main/org/nlogo/prim/etc/_mean.java

```diff
@@ -41,8 +41,8 @@ public double report_1(Context context, LogoList list) throws LogoException {
             I18N.errorsJ().getN("org.nlogo.prim._mean.cantFindMeanOfNonNumbers",
                 Dump.logoObject(elt), TypeNames.name(elt)));
       }
-      sum += ((Double) elt).doubleValue();
+      mean += ((Double) elt).doubleValue() / list.size();
```

Comment:
You could try storing `list.size()` in a variable above and dividing by that. Although `list.size()` is fast (it resolves to [`Vector.length`](https://github.com/scala/scala/blob/v2.11.7/src/library/scala/collection/immutable/Vector.scala#L81)), tossing that extra math operation and method calls might reduce the hit (though I'm a little surprised this isn't jitted away).